tests/e2e/utils/cluster.py

```diff
@@ -4,21 +4,25 @@
 import subprocess
 
 
-def run_oc(args: list[str]) -> subprocess.CompletedProcess:
+def run_oc(args: list[str], input=None) -> subprocess.CompletedProcess:  # noqa: A002
     """Run a command in the OpenShift cluster."""
     try:
         res = subprocess.run(  # noqa: S603
             ["oc", *args],  # noqa: S607
             capture_output=True,
             text=True,
             check=True,
+            input=input,
         )
         return res
     except subprocess.CalledProcessError as e:
-        print(
-            f"Error running oc command {args}: {e}, stdout: {e.output}, stderr: {e.stderr}"
-        )
-        raise
+        if "AlreadyExists" in e.stderr:
```

Comment:
IMHO it need to be checked outside that function. As this one just runs any command with any output, where "AlreadyExists" (raise) might even be expected outcode